### DISCUSSION TITLE
XCOM 2 WOTC Crashing consistently

### DISCUSSION THREAD

**Discussion by EcholonX** [X2CommunityCore/xcom2-launcher#274] [Help]
Started a new campaign a few weeks ago and after awhile the game had finally broken down. I was hoping to continue replaying through with this new AML launcher hoping it would fix some issues concerning my mod count (200 something). I did everything I was used to, regenerating inis, starting with no mods and restarting with them; nothing worked. And then I found this error file, can anyone explain this and help me work out this error?
[error.log](https://github.com/X2CommunityCore/xcom2-launcher/files/6632491/error.log)

**Comment by 1Kr0noZ1** [comment]
Sounds like loading all your mods makes your PC run out of memory within the amount that the game can actually reserve.
How much RAM do you have? I had issues like that with other games (Cities Skyline and Kerbal Space Program) when loading up a ton of mods.

**Comment by Dragon-32** [comment]
If the Alternate Mod Launcher starts XCOM 2 and it's that which crashes, then AML has done its job of being a launcher and your problem is with the mods you are using and not the launcher.

As 1Kr0noZ1 says, that error.log indicates you don't have enough RAM.

For general information, if you haven't already seen it, this is some great mod troubleshooting advice:
https://www.reddit.com/r/xcom2mods/wiki/mod_troubleshooting

**Comment by Hulkninja** [comment]
Error log clearly states your system ran out of memory. Either upgrade your RAM, or remove some of the mods. 
This seems unrelated to the launcher.

**Comment by RevZero** [comment]
Moving to discussions.
